doc/classes/Basis.xml

```diff
@@ -98,6 +111,18 @@
 			<return type="Vector3" />
 			<description>
 				Assuming that the matrix is the combination of a rotation and scaling, return the absolute value of scaling factors along each axis.
+				[codeblock]
+				var my_basis = Basis(
+				    Vector3(2, 0, 0),
+				    Vector3(0, 4, 0),
+				    Vector3(0, 0, 8)
+				)
+				# Rotating the Basis in any way preserves its scale.
+				my_basis = my_basis.rotated(Vector3.UP, PI)
+				my_basis = my_basis.rotated(Vector3.RIGHT, PI / 2)
```

Comment:
I would suggest using TAU in code examples instead of PI since it results in more readable code.

```suggestion
				my_basis = my_basis.rotated(Vector3.UP, TAU / 2)
				my_basis = my_basis.rotated(Vector3.RIGHT, TAU / 4)
```

This can be read as "perform a half turn around UP and a quarter turn around RIGHT".

Reply:
I love TAU myself, but I decided to use PI because it's more commonly understood. PI is a must when learning about angles. I could still change it if it would make the code examples more consistent across the board.